primap2/pm2io/_data_reading.py

```diff
@@ -139,6 +163,10 @@ def convert_long_dataframe_if(
         in the interchange format.
         Default: "%F", i.e. the ISO 8601 date format.
 
+    convert_nan : bool, optional
+        If set to true, string values in the data columns as listed in NA_VALUES will be
```

Comment:
For the user, NA_VALUES is not readily available, so the documentation here is rather unclear. Either just use something like "common values denoting NaN" or export the NA_VALUES and add the full name (`primap2.pm2io.NA_VALUES` or so) or maybe even make it possible to supply an own list and use NA_VALUES only as a default.

Reply:
A user defined list would be best. Especially with the option to define what is mapped to 0 and what is mapped to NA (and a default conversion listed explicitly in the docs).

Reply:
See discussion of string conversion above. A user controllable conversion has been implemented now.